docs/advanced_guide.rst

```diff
@@ -2,20 +2,41 @@
 
 Advanced Tutorials
 ==================
+This section contains examples and tutorials on more advanced topics
```

Comment:
A short explanation of what makes these advanced would be nice, even for beginner users. That way they have a sense what advanced usage means and if they even need to be advanced

Here's a suggestion

```suggestion
This section contains examples and tutorials on more advanced topics, such as Multi Core computation, Custom operations, and more in depth applications
```